### ISSUE TITLE
LSTM model parameters

### ISSUE THREAD

**Issue by diamond-jlu** [dhopp1/nowcast_lstm#12]
Dear Professor Daniel Hopp, when I used your LSTM model code to make prediction, the RMSE I got was about 45,000, which was much larger than the ARMA model. I used the weekly degree data and daily degree data to predict the weekly degree data. May I ask which parameters in the LSTM model should I modify to improve the prediction accuracy? I would appreciate it if you could read and answer my questions carefully!

**Comment by dhopp1** [comment]
Hello, if your RMSE is so large, one likely reason is that the data need to be scaled to be aorund the same order of magnitude for the LSTM to work optimally. That means either convert the data to growth rates, [min max scale](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html) it, normalize, it, etc.

As an example, say you are predicting total trade using the oil price and volume of oil traded. You're data might look like:
- Trade: 4000 (millions of USD)
- oil price: 80 (USD)
- volume of oil traded: 8000000 (barrels of oil)

These units all have quite different scales, and the LSTM likely will not perform well. Rather convert them to look something like:
- Trade: 0.05
- oil price: 0.04
- volume of oil traded: 0.12

Now the variables have similar scales and the LSTM will probably perform better.

Otherwise you need to hyperparameter tune, you can find some values to help you out in [this](https://github.com/dhopp1/nowcasting_benchmark/blob/main/html_output/model_lstm.html) notebook, the section titled "Below is a list of recommended hyperparameter values to use for grid search with the LSTM if results are not very good with default parameters"
